src/utils/init.ts

```
@@ -3,22 +3,45 @@ import { AppOptions } from "index";
 
 import { EnvConfig, initEnv } from "./env";
 import { ExtendedPrismaClient, initPrisma } from "./prisma";
+import * as Sentry from "@sentry/node";
+import { initSentry } from "./sentry";
+import { randomUUID } from "crypto";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)